diff --git a/feeds/luci/applications/luci-app-usb-printer/luasrc/controller/usb_printer.lua b/feeds/luci/applications/luci-app-usb-printer/luasrc/controller/usb_printer.lua
index 9afc109..6f3df8c 100644
--- a/feeds/luci/applications/luci-app-usb-printer/luasrc/controller/usb_printer.lua
+++ b/feeds/luci/applications/luci-app-usb-printer/luasrc/controller/usb_printer.lua
@@ -21,10 +21,10 @@ function index()
 		return
 	end
 	
-  entry({"admin", "nas"}, firstchild(), "NAS", 44).dependent = false
+  entry({"admin", "services"}, firstchild(), "Services", 44).dependent = false
 	
 	local page
 
-	page = entry({"admin", "nas", "usb_printer"}, cbi("usb_printer"), _("USB Printer Server"), 50)
+	page = entry({"admin", "services", "usb_printer"}, cbi("usb_printer"), _("USB Printer Server"), 50)
 	page.acl_depends = { "luci-app-usb-printer" }
 end
