diff --git a/feeds/luci/applications/luci-app-openvpn/Makefile b/feeds/luci/applications/luci-app-openvpn/Makefile
index 4dfcc68..25318f1 100644
--- a/feeds/luci/applications/luci-app-openvpn/Makefile
+++ b/feeds/luci/applications/luci-app-openvpn/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 LUCI_TITLE:=LuCI Support for OpenVPN
-LUCI_DEPENDS:=+luci-compat
+LUCI_DEPENDS:=+luci-compat +openvpn-openssl +openvpn-easy-rsa +kmod-tun
 
 PKG_LICENSE:=Apache-2.0
 
